app/Resources/views/base.html.twig

```diff
@@ -155,6 +163,12 @@
                 $(document).ready(function() {
                     hljs.initHighlightingOnLoad();
                 });
+
+                $(function() {
```

Comment:
Could we move this JS code to the above `$(document).ready(function() { ... }` block?